css/store.css

```diff
@@ -0,0 +1,176 @@
+.main{
+    display: flex;
+    width: 100%;
+    height: 1235px;
+    padding-top: 90px;
+
+}
+
+.form{
+    width: 50%;
+    padding: 40px;
+    color: #666666;
+}
+
+.cover{
+    width: 50%;
+    background-repeat: no-repeat;
+    background-image:url(../img/store-image_by-andrew-neel-unsplash.jpg);
+    background-size:cover;
+}
+
+/* FONTS */
+h1{
+    font-weight:500;
+    font-size: 50px;
+    color: var(--orange);
+    padding: 50px 0 59px 0;
+}
+h5{
+    font-size: 18px;
+    color:var(--grey-light);
+    margin-left: 10px;
+}
+
+
+/* FORM FIELDS */
+fieldset{
+    border: none;
+}
+input[type=text], select, option.field-flex-item {
+  width: 100%;
+  padding: 18px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 2px solid #666666;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.field-flex{
+    display: flex;
+    justify-content:space-evenly;
+    width: 100%;
+}
+
+.field-fname{
+    width: 50%;
+    margin-right: 10px;
+}
+.field-lname{
+    width: 50%;
+}
+
+.field-fcity{
+    width: 100%;
+    margin-right: 10px;
+}
+input[id=fpostcode] {
+    height: 61px;
+}
+
+
+/* FAKE RADIO */
+label > input[type="radio"] {
+  display: none;
+}
+label > input[type="radio"] + *::before {
+  content: "";
+  display: inline-block;
+  vertical-align:middle;
+  width: 20px;
+  height: 20px;
+  margin-right: 10px;
+  border-radius: 50%;
+  border-style: solid;
+  border-width: 2px;
+  border-color: gray;
+  box-sizing: border-box;
+}
+
+label > input[type="radio"]:checked + *::before {
+  background: radial-gradient(#DE6E49 0%, #DE6E49 50%, transparent 60%);
+  border-color: grey;
+}
+
+label{
+    color: gray;
+    font-weight: 500;
+    font-size: 19px;
+}
+
+.label400{
+    color: gray;
+    font-weight: 300;
+    font-size: 21px;
+}
+
+.radio-span{
+    margin-left: 20px;
+}
+
+
+
+/* /BUTTONS */
+button{
+    width: 269px;
+    height: 70px;
+    font-size: 22px;
+}
+
+
+input[type="checkbox"]{
+  
+  width: 20px;
+  height: 20px;
+}
+
+/* FAKE CHECK BOX */
+[type="checkbox"]{
+    opacity:0;
+}
+[type="checkbox"] + label{
+    position: relative;
+    padding-left: 20px;
+    cursor: pointer;
+    display: inline-block;
+    color:#DE6E49;
+    margin-left:-15px;
+    height: 0px;
+}
+
+[type="checkbox"] + label ::before{
+    content:"";
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 20px;
+    height: 20px;
+    outline: 2px solid gray;
+    border-radius: 4px;
+    background: white;
+
+}
+
+[type="checkbox"]:checked + label ::before{
+    content:"";
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 20px;
+    height: 20px;
+    background-image:url(../img/tick_checkbox.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 15px;
+}
+ /* LINK CHECKBOX */
+.terms > .link{
+    color: #DE6E49;
+    display: inline-block;
+    font-size: 18px;
+}
+.terms{
+    display: flex;
+    margin-top: 40px;
+}
```

Comment:
Good job separating the store page styles into a different file! How could you separate the styles for the homepage (index.hml) into a separate file so that `style.css` only contains share styles needed on both pages?